src/app/docs/kagent/resources/release-notes/page.mdx

```diff
@@ -0,0 +1,334 @@
+---
+title: "Release Notes"
+pageOrder: 3
+description: "Review the release notes for kagent."
+---
+
+import { Tabs, Tab } from '@/components/mdx/tabs';
+
+export const metadata = {
+  title: "Release Notes",
+  description: "Review the release notes for kagent.",
+  author: "kagent.dev"
+};
+
+# Release Notes
+
+The kagent documentation shows information only for the latest release. If you run an older version, review the release notes to understand the main changes from version to version.
+
+# v0.6
+
+Review the main changes from kagent version 0.5 to v0.6, then continue reading for more detailed information.
+
+* A new Helm chart for kmcp CRDs is available.
+* API string references to resources in other namespaces in the format `namespace/name` now fail. Instead, the APIs have a separate field for you to specify the namespace of the resource.
+* The Tools API moves or eliminates some APIs entirely in favor of new kmcp APIs.
+* The Agents APIs now require a top-level `type` field to support the new BYO agent functionality.
+* The ModelConfig APIs rename the secret name field from `apiKeySecretRef` to `apiKeySecret`.
+
+## New! Helm chart for kmcp CRDs
+
+Previously, the kagent CRD Helm chart installed the CRDs for both kagent and kmcp. Now, the kmcp CRDs are installed in a separate Helm chart.
+
+* kagent CRD Helm chart location (no change): `oci://ghcr.io/kagent-dev/kagent/helm/kagent-crds`
+* kmcp CRD Helm chart location: `oci://ghcr.io/kagent-dev/kmcp/helm/kmcp-crds`
+
+## General API changes
+
+API string references to resources in other namespaces in the format `namespace/name` now fail. Instead, the APIs have a separate field for you to specify the namespace of the resource.
+
+## Tools APIs
+
+The Tools-related APIs are split up into several different APIs. Some functionality is moved to kmcp, such as the ToolServer API.
+
+### ToolServer
+
+The ToolServer API is completely removed from kagent. Instead, use [kmcp APIs](https://kagent.dev/docs/kmcp) to create and manage tools.
+
+#### Stdio ToolServer now in kmcp MCPServer
+
+Flip through the following tabs to understand the API differences between kagent ToolServer and kmcp MCPServer for the Stdio transport type.
+
+<Tabs tabs={[
```

Comment:
Also mention the `RemoteMCPServer` and `Service`. It's not completely accurate to say that the functionality moved wholesale into `kmcp`